core/bcast/bcast.go

```diff
@@ -93,6 +95,24 @@ func (b Broadcaster) Broadcast(ctx context.Context, duty core.Duty,
 	case core.DutyRandao:
 		// Randao is an internal duty, not broadcasted to beacon chain
 		return nil
+	case core.DutyVoluntaryExit:
+		// JSON decoding from the previous component
+		ve := new(eth2p0.SignedVoluntaryExit)
+		err := json.Unmarshal(aggData.Data, ve)
+		if err != nil {
+			return errors.Wrap(err, "json decoding voluntary exit")
+		}
```

Comment:
Please follow above pattern of extracting this to `core.DecodeExitAggSignedData`

Reply:
Done